subprojects/dependency-management/src/main/java/org/gradle/api/internal/attributes/EmptySchema.java

```diff
@@ -104,6 +105,12 @@ public List<Attribute<?>> getAttributeDisambiguationPrecedence() {
         return Collections.emptyList();
     }
 
+    @Nullable
+    @Override
+    public Attribute<?> getAttributeByName(String name) {
+        return null;
```

Comment:
Should this throw too?

Reply:
No, we expect this to be called from within `DefaultAttributeSelectionSchema`.